docs/upgrading/android-csproj.md

```diff
@@ -0,0 +1,350 @@
+---
+title: "Xamarin.Android csproj conversion"
+description: ""
+ms.date: 10/01/2022
+---
+
+# Xamarin.Android csproj conversion
+
+A .NET 6 project for a Xamarin.Android application looks something
+like:
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net6.0-android</TargetFramework>
+    <OutputType>Exe</OutputType>
+  </PropertyGroup>
+</Project>
+```
+
+For a "library" project, omit the `$(OutputType)` property
+completely or specify `Library`.
+
+## Bindings projects
+
+Changes for bindings projects are documented [here][binding].
+
+[binding]: OneDotNetBindingProjects.md
```

Comment:
Does this link work?

We could bring this doc over as well: https://github.com/xamarin/xamarin-android/blob/main/Documentation/guides/OneDotNetBindingProjects.md